FBSnapshotTestCase/FBSnapshotTestController.m

```diff
@@ -226,6 +228,15 @@ - (NSString *)_fileNameForSelector:(SEL)selector
   if (0 < identifier.length) {
     fileName = [fileName stringByAppendingFormat:@"_%@", identifier];
   }
+  
+  if (self.isDeviceAgnostic) {
+    UIDevice *device = [UIDevice currentDevice];
+    NSString *model  = [device.model stringByReplacingOccurrencesOfString:@" " withString:@"_"];
+    NSString *os     = [device.systemVersion stringByReplacingOccurrencesOfString:@"." withString:@"_"];
```

Comment:
Haven't thought about it but is definitely better.
Will add that to my list of things to do tomorrow.

:bow: